pure.zsh

```diff
@@ -47,6 +47,15 @@ prompt_pure_check_cmd_exec_time() {
 	(($elapsed > ${PURE_CMD_MAX_EXEC_TIME:=5})) && prompt_pure_human_time $elapsed
 }
 
+prompt_pure_clear_screen() {
+	# enable output to terminal
+	zle -I
+	# clear screen and move cursor to (0, 0)
+	print -n "\e[2J\e[0;0H"
```

Comment:
Ok. Let's stick with single-quotes wherever possible for now then. We can always change later if that feels weird.